spring-cloud-dataflow-parent/pom.xml

```diff
@@ -113,6 +114,22 @@
 				<artifactId>spring-security-oauth2-client</artifactId>
 				<version>${spring-security.version}</version>
 			</dependency>
+			<!-- Override Logback provided by Spring Boot -->
```

Comment:
Because we do NOT inherit from SBD all we have to do is declare the dep. mgmt. above the SBD dep. mgmt ([background](https://docs.spring.io/spring-boot/docs/2.7.18/maven-plugin/reference/htmlsingle/#using.import)).